### ISSUE TITLE
HDv3模型的复现

### ISSUE THREAD

**Issue by ZFU123456** [hzwer/ECCV2022-RIFE#336]
flow, mask, merged = self.flownet(torch.cat((imgs, gt), 1), scale=scale, training=training)
        **loss_l1** = (merged[2] - gt).abs().mean()
        **loss_smooth** = self.sobel(flow[2], flow[2]*0).mean()
        # loss_vgg = self.vgg(merged[2], gt)
        if training:
            self.optimG.zero_grad()
            **loss_G = loss_cons + loss_smooth * 0.1**
            loss_G.backward()
            self.optimG.step()
        else:
            flow_teacher = flow[2]
        return merged[2], {
            'mask': mask,
            'flow': flow[2][:, :2],
            'loss_l1': loss_l1,
            'loss_cons': loss_cons,
            'loss_smooth': **loss_smooth,**
            }
想问您使用的是几个loss？是”loss_l1+loss_cons+loss_smooth“三个loss吗？还是仅仅loss_cons + loss_smooth * 0.1？
还想问下作者，使用HDv3复现插多帧模型的时候，训练并不成功，模型的psnr值为2.多，是什么原因呢？

**Comment by hzwer** [comment]
Hi，我用的是 loss_l1+loss_cons*0.01+loss_vgg(可选)；psnr这么低的情况下，是不是有可能数据喂错了
你可以只用 l1 loss跑通再试别的

**Comment by apple1986** [comment]
> loss_cons

请问这里的loss_cons是什么？谢谢

**Comment by hzwer** [comment]
@apple1986 是一个 student 向 teacher 学习光流的 loss

**Comment by xjjww** [comment]
> @apple1986 是一个 student 向 teacher 学习光流的 loss
怎么没有teacher网络输出和gt之间的loss？

**Comment by hzwer** [comment]
@xjjww 
... 可能这版本偷懒加在 loss_cons 里面了
